bidswitch-json/src/main/scala/com/powerspace/openrtb/bidswitch/bidrequest/BidSwitchDealSerde.scala

```diff
@@ -13,8 +13,8 @@ object BidSwitchDealSerde extends ConfiguredSerde {
   import io.circe._
   import io.circe.generic.extras.semiauto._
 
-  implicit val dealExtEncoder: Encoder[DealExt] = deriveEncoder[DealExt].cleanRtb
+  implicit val dealExtEncoder: Encoder[DealExt] = deriveConfiguredEncoder[DealExt].cleanRtb
 
-  val dealExtDecoder: Decoder[DealExt] = deriveDecoder[DealExt]
+  val dealExtDecoder: Decoder[DealExt] = deriveConfiguredDecoder[DealExt]
```

Comment:
it's been renamed or it gives different functionality?

Reply:
It has been renamed to avoid confusion between `io.circe.generic.extras.semiauto` and `io.circe.generic.semiauto`